### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Menduist**

### CODE REVIEW

**Comment by arnetheduck** [comment]
while you're here, the contributor/maintainer section needs refreshing

**Comment by Menduist** [comment]
Yeah, I'm actually considering removing it entirely, github does a good job of showing them [here](https://github.com/status-im/nim-libp2p/graphs/contributors) anyway
Or just adding
https://contrib.rocks/image?repo=status-im/nim-libp2p&max=12

File: README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
   <a href="https://libp2p.io"><img width="250" src="./.assets/full-logo.svg?raw=true" alt="nim-libp2p logo" /></a>
 </h1>
 
-<h3 align="center">The Nim implementation of the libp2p Networking Stack.</h3>
+<h3 align="center">The <a href="https://nim-lang.org/">Nim</a> implementation of the <a href="https://libp2p.io/">libp2p</a> Networking Stack.</h3>
 
 <p align="center">
 <a href="https://github.com/status-im/nim-libp2p/actions"><img src="https://github.com/status-im/nim-libp2p/actions/workflows/ci.yml/badge.svg" /></a>
-<a href="https://codecov.io/gh/status-im/nim-libp2p"><img src="https://codecov.io/gh/status-im/nim-libp2p/branch/master/graph/badge.svg?token=UR5JRQ249W"/></a>
+<a href="https://codecov.io/gh/status-im/nim-libp2p"><img src="https://codecov.io/gh/status-im/nim-libp2p/branch/unstable/graph/badge.svg?token=UR5JRQ249W"/></a>
```
Comment (diegomrsantos) [inline README.md:9]: Why unstable? Additionally, the link opens the master branch.

File: README.md [hunk 2]
```diff
@@ -63,25 +61,28 @@ List of packages modules implemented in nim-libp2p:
...
+| [libp2p-tor](libp2p/transports/tortransport.nim)           | Tor Transport                                                                                                    |
 | **Secure Channels**                                        |                                                                                                                  |
-| [libp2p-secio](libp2p/protocols/secure/secio.nim)          | [Secio](https://docs.libp2p.io/concepts/protocols/#secio) secure channel                                         |
-| [libp2p-noise](libp2p/protocols/secure/noise.nim)          | [Noise](https://github.com/libp2p/specs/tree/master/noise) secure channel                                        |
-| [libp2p-plaintext](libp2p/protocols/secure/plaintext.nim)  | [Plain Text](https://github.com/libp2p/specs/tree/master/plaintext) for development purposes                     |
+| [libp2p-secio](libp2p/protocols/secure/secio.nim)          | Secio secure channel                                                                                             |
```
Comment (diegomrsantos) [inline README.md:73]: Are those whitespaces?

File: README.md [hunk 3]
```diff
@@ -63,25 +61,28 @@ List of packages modules implemented in nim-libp2p:
...
+| [libp2p-yamux](libp2p/muxers/yamux/yamux.nim)              | [Yamux](https://docs.libp2p.io/concepts/multiplex/yamux/) multiplexer                                            |
 | **Data Types**                                             |                                                                                                                  |
-| [peer-id](libp2p/peerid.nim)                               | [Cryptographic identifiers](https://docs.libp2p.io/concepts/peer-id/)                                            |
-| [peer-store](libp2p/peerstore.nim)                         | ["Phone book" of known peers](https://docs.libp2p.io/concepts/peer-id/#peerinfo)                                 |
+| [peer-id](libp2p/peerid.nim)                               | [Cryptographic identifiers](https://docs.libp2p.io/concepts/fundamentals/peers/#peer-id)                         |
+| [peer-store](libp2p/peerstore.nim)                         | ["Phone book" of known peers](https://docs.libp2p.io/concepts/fundamentals/peers/#peer-store)                    |
```
Comment (diegomrsantos) [inline README.md:81]: Maybe "Address book"?

File: README.md [hunk 4]
```diff
@@ -114,6 +115,7 @@ Clone and Install dependencies:
 git clone https://github.com/status-im/nim-libp2p
 cd nim-libp2p
 nimble install -dy
+# or use `nimble install_pinned` to install the dependencies versions used in CI
```
Comment (diegomrsantos) [inline README.md:118]: How about "or use the custom task..."

File: README.md (hunk 2 above)
Comment (Menduist) [reply README.md:73]: Yeah, it looks good in ascii too :)

**Review by diegomrsantos — APPROVED** [review]

**Pull request merged**